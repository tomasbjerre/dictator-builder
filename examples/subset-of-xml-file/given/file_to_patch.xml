<rootelement>
    <a>b</a>
    <c>d</c>
    <e>f</e>
</rootelement>