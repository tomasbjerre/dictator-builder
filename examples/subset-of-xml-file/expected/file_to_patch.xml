<rootelement>
    <a>1</a>
    <c>d</c>
    <e>f</e>
    <g>2</g>
</rootelement>